templates/email/default/problem-confirm.html

```diff
@@ -17,6 +17,8 @@ <h1 style="[% h1_style %]">Please confirm your&nbsp;report</h1>
 Please note that [% c.cobrand.council_name %] is not responsible for this type
 of problem, so it will instead be sent to [% report.body %].
 [% END %]
+
+[% INCLUDE '_problem-confirm_extra.html' %]
```

Comment:
If you use TRY / INCLUDE / CATCH file / END here (see examples elsewhere) you won't need to have the blank files under `default/`.

Reply:
ah, for some reason I had it in my head that empty files were better than TRY / INCLUDE / CATCH. I have no idea why :|